File: Programmers/위장(hash lv2).py
//Solve
def solution(clothes):
    inventory = {}
    answer = 1
    for cloth in clothes:
        if cloth[1] in inventory:
            inventory[cloth[1]] +=1
        else:
            inventory[cloth[1]] =1

    for i, v in inventory.items():
        answer *= (v+1)

    return answer-1

File: Programmers/코딩테스트 고득점 Kit/Hash/위장(hash lv2).js
function solution(clothes) {
    let answer = 1;
    const map = new Map();
    for (let cloth of clothes){
        const kind=cloth[1]
        const count=map.get(kind)
        if(count){
            map.set(kind, count + 1);
        }else{
            map.set(kind, 1);
        }
    }

    for (let value of map.values()){
        answer *= (value + 1);
    }
    return answer-1;
}